test/_btr3-transactiondetail-examples.json

```diff
@@ -90,7 +90,7 @@
                 "detailFundsType": "",
                 "bankReferenceNumber": "AGN",
                 "customerReferenceNumber": "935892",
-                "detailText": "POINT OF SALE"
+                "detailText": "POINT OF SALE/"
```

Comment:
I make a few changes to the test cases, because I believe the grammar was implemented incorrectly.

The specification states that 

> Therefore, the end of a record ending in text cannot be delimited using these characters. Instead, the logical record continues until another record that is not a continuation (88) record begins

![Screen Shot 2021-03-02 at 3 57 32 PM](https://user-images.githubusercontent.com/9813258/109714848-e8f73200-7b70-11eb-90d2-27d6723683c8.png)

This means that if there is a `/` at the end of a line, it should be considered part of the text.